### ISSUE TITLE
signed responses?

### ISSUE THREAD

**Issue by jonasschnelli** [bitpay/bitauth#39]
Maybe i don't see it in the code, but it looks like that responses are not signed at all?
How would this prevents a MITM to temper a response?

**Comment by braydonf** [comment]
Yep, requests are signed whereas the responses depend on the security of TLS/X509 certificates currently.
